profiles/coreos/base/make.defaults

```diff
@@ -35,7 +35,7 @@ USE="${USE} -openmp"
 POLICY_TYPES="targeted mcs mls"
 
 # Disable packages or optional features with distribution issues.
-ACCEPT_RESTRICT="* -bindist -mirror"
```

Comment:
Should this be on `profiles/coreos/sdk/make.defaults` instead?

Reply:
This changes it everywhere.  We could lift the restriction only in the SDK since cargo shouldn't be installed anywhere else, but is there a reason to?  It looked like the intent of restricting this originally was just to get our attention.  There doesn't seem to be any affect on what actually gets mirrored.